set.mm

```diff
@@ -22426,6 +22430,63 @@ primitive connectives (no defined terms) on the right-hand side.
       FBTUDCAUCJKLUDBCMNOABCPRS $.
   $}
 
+  $( Absorption of existence condition by uniqueness.  (Contributed by NM,
+     4-Nov-2002.)  Shorten proof and avoid ~ df-eu .  (Revised by BJ,
+     14-Oct-2022.) $)
+  moabs $p |- ( E* x ph <-> ( E. x ph -> E* x ph ) ) $=
+    ( wmo wex wi ax-1 wn nexmo pm2.6 ax-mp impbii ) ABCZABDZLEZLMFMGLENLEABHMLI
+    JK $.
+
+  $( Any proposition holds for some ` x ` or holds for at most one ` x ` .
+     (Contributed by NM, 8-Mar-1995.)  Shorten proof and avoid ~ df-eu .
+     (Revised by BJ, 14-Oct-2022.) $)
+  exmo $p |- ( E. x ph \/ E* x ph ) $=
+    ( wex wmo wo wn wi nexmo df-or mpbir ) ABCZABDZEKFLGABHKLIJ $.
+
+  ${
+    $d x z $.  $d y z $.  $d z ph $.  $d z ps $.
+    nfmod2.1 $e |- F/ y ph $.
+    nfmod2.2 $e |- ( ( ph /\ -. A. x x = y ) -> F/ x ps ) $.
+    $( Bound-variable hypothesis builder for the at-most-one quantifier.
+       (Contributed by Mario Carneiro, 14-Nov-2016.)  Avoid ~ df-eu .  (Revised
+       by BJ, 14-Oct-2022.) $)
+    nfmod2 $p |- ( ph -> F/ x E* y ps ) $=
+      ( vz wmo weq wi wal wex df-mo nfv wn wa wnf nfeqf1 adantl nfimd nfald2
+      nfexd nfxfrd ) BDHBDGIZJZDKZGLACBDGMAUFCGAGNAUECDEACDICKOZPBUDCFUGUDCQACD
+      GRSTUAUBUC $.
```

Comment:
I don't see an advantage of this proof over the old one.  See orim12d/ALT for a similar situation.  This is not a consistent view.